### PR TITLE
Course Discovery default db is setup for docker.

### PR DESCRIPTION
**Pull request by feanil**
Sandboxes don't have a 'db' host like docker will.  They need to use 'localhost' for mysql.

### CODE REVIEW

**Comment by feanil** [comment]
@edx/devops I verified that this fixes CI.

**Comment by feanil** [comment]
@cpennington FYI

File: docker/build/course_discovery/ansible_overrides.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
-course_discovery_gunicorn_host: 0.0.0.0
\ No newline at end of file
+course_discovery_gunicorn_host: 0.0.0.0
+COURSE_DISCOVERY_MYSQL: 'localhost'
```
Comment (e0d) [inline docker/build/course_discovery/ansible_overrides.yml]: this should be db

**Comment by e0d** [comment]
:+1:

**Pull request merged**